### DISCUSSION TITLE
Install on D: drive

### DISCUSSION THREAD

**Discussion by RazaelDAbyss** [o3de/o3de#5921] [Q&A]
Anyone know how to get the Asset Processor to work from the D: drive.  Dell partitioned my hard drive badly, I have 200Gb on C: and 1.8Tb on D:.  C: is taken up with VS and the SDKs, only room I have left is on the D: drive.  The editor and project seemed to install fine, but when I start the Asset Processor is says that the D: location is not a valid Project location!

**Comment by lmbr-pip** [comment]
Can you clarify the steps taken:
* Are you building from source or an installer? If the installer where did you install?
* Can you provide actual error message from AP?
